swift/ql/lib/change-notes/2023-09-13-array-content-unification.md

```diff
@@ -0,0 +1,5 @@
+---
+category: breaking
```

Comment:
With "ArrayContent" mapping to "CollectionContent", is this really breaking?

@MathiasVP I'd like to hear your opinion on this matter as well.

Reply:
No, if we keep ArrayContent around as a deprecated alias for CollectionContent we can probably just add a deprecation change note

Reply:
There's two separate potentially-breaking things here - the `ArrayContent` type and the `"ArrayElement"` string in models-as-data. Right now I'm breaking the former (probably not actually used in the wild yet, given that it's fairly tricky to use and quite recently added) and not the latter.

Reply:
Can't we just write `deprecated class ArrayContent = CollectionContent` and avoid the breaking change altogether?

I do agree that, since we're not really expecting people to be writing custom queries yet, it's probably not a big deal to break it.